src/editor/components/AddBlock/AddBlock.scss

```diff
@@ -23,8 +23,8 @@ $block: '.#{$ns}add-block';
         width: $buttonWidth;
         height: $buttonHeight;
 
-        color: var(--g-color-text-inverted-primary);
-        background-color: var(--pc-color-base-neon);
+        color: var(--g-color-text-primary);
+        background-color: var(--pc-editor-base-color);
```

Comment:
why don't we use colors from uikit for editing?

Reply:
`--pc-editor-base-color` contains color form uikit, I just created this variable for comfortable working with editor colors